components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -911,13 +914,23 @@ public List<Property> getConfigurationProperties() {
         userIdLocation.setDisplayOrder(7);
         configProperties.add(userIdLocation);
 
+        Property scopes = new Property();
+        scopes.setName(OIDCAuthenticatorConstants.SCOPES);
+        scopes.setDisplayName("Scopes");
+        scopes.setRequired(false);
+        scopes.setDescription("List of scopes. Must be space-separated and at least include 'openid'");
```

Comment:
mandatorily include openid
remove "at least include openid" part
say "A space-separated list of scopes"